storage/replica_raftstorage.go

```diff
@@ -351,7 +351,10 @@ func (r *Replica) updateRangeInfo() error {
 	// Load the system config.
 	cfg := r.rm.Gossip().GetSystemConfig()
 	if cfg == nil {
-		return fmt.Errorf("no system config available")
+		// This could be before the system config was ever gossiped,
+		// or it expired. Let the gossip callback set the info.
+		log.Warningf("no system config available, cannot determine range MaxBytes")
```

Comment:
can you make this an `errors.New` and check equality in the caller? That would make me feel better than returning `nil`.